app/javascript/widget/App.vue

```diff
@@ -134,10 +134,20 @@ export default {
         });
       });
     },
-    setLocale(locale) {
+    setLocale(localeWithVariation) {
       const { enabledLanguages } = window.chatwootWebChannel;
-      if (enabledLanguages.some(lang => lang.iso_639_1_code === locale)) {
-        this.$root.$i18n.locale = locale;
+      const localeWithoutVariation = localeWithVariation.split('_')[0];
+      const hasLocaleWithoutVariation = enabledLanguages.some(
+        lang => lang.iso_639_1_code === localeWithoutVariation
+      );
+      const hasLocaleWithVariation = enabledLanguages.some(
+        lang => lang.iso_639_1_code === localeWithVariation
+      );
+
+      if (hasLocaleWithVariation) {
+        this.$root.$i18n.locale = localeWithVariation;
+      } else if (hasLocaleWithoutVariation) {
+        this.$root.$i18n.locale = localeWithoutVariation;
       }
```

Comment:
I'm not sure if I understand this change. Why do we split the locale here? Also, I don't think it is ideal to use pt as a substitute for pt_BR or zh instead of zh_TW.

Reply:
@sojan-official Thoughts?

Reply:
We have `pt` and `pt_BR`. With this change, if `pt_BR` is not available it'll show `pt`. If not correct, we could do a hard check for `pt_BR`.